src/places/dto/updatePlaceDto.ts

```diff
@@ -1,20 +1,20 @@
-import { IsNotEmpty, IsNumber, IsOptional, IsString } from 'class-validator';
+import { IsNumber, IsOptional, IsString } from 'class-validator';
 
 export class UpdatePlaceDto {
   @IsOptional()
-  @IsNotEmpty()
+  @IsString()
   name?: string;
 
   @IsOptional()
-  @IsNotEmpty()
```

Comment:
Does it work without IsNotEmpty to prevent blank strings?

Reply:
Good point.